crowbar_framework/Gemfile

```diff
@@ -58,7 +58,7 @@ gem "rack-mini-profiler", "~> 0.9",
 
 group :development, :test do
   gem "brakeman", "~> 2.6"
-  gem "rspec-rails", "~> 2.14"
+  gem "rspec-rails", "~> 3.1"
```

Comment:
It is similar. and i have tested all the version constraints over weeks pretty hard.